### PR TITLE
OCLOMRS-356: Implement search on bulk add concepts

### PR DESCRIPTION
**Pull request by ozezzy**
# JIRA TICKET NAME:
[Implement search on bulk add concepts](https://issues.openmrs.org/browse/OCLOMRS-356)

# Summary:
A user should be able to search concepts and populate the text area with IDs to bulk add to a dictionary

### CODE REVIEW

**Comment by abulojoshua1** [comment]
@ozezzy, kindly fix the coverage drop issue.

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx [hunk 1]
```diff
@@ -18,81 +18,70 @@ const match = {
...
+  it('can search for and select "Bronze ..."', () => {
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+    const input = wrapper.find('#search');
+    input.simulate('change', { target: { value: '' } });
+    input.simulate('keydown', { key: 'ArrowDown' });
+    input.simulate('keydown', { key: 'Enter' });
```
[the remark this answers is not in the document]
Comment (ozezzy) [reply src/tests/bulkConcepts/addBulkConcepts.test.jsx]: done

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx [hunk 2]
```diff
@@ -18,81 +18,70 @@ const match = {
...
-    expect(mapStateToProps(initialState).cielConcepts).toEqual(['1']);
+  it('simulate textarea text change', () => {
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+    const input = wrapper.find('#idsText').at(0);
+    input.simulate('change', { target: {value: 'Bronze' } });
+    input.simulate('keydown', { key: 'Enter' });
```
[the remark this answers is not in the document]
Comment (ozezzy) [reply src/tests/bulkConcepts/addBulkConcepts.test.jsx]: done

**Comment by ozezzy** [comment]
> @ozezzy, kindly fix the coverage drop issue.

The line coverage for changed lines is 100%, I think the drop is due to removed snippets.

File: src/components/bulkConcepts/bulkConceptList.jsx [hunk 3]
```diff
@@ -1,42 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {
-  Form,
-  FormGroup,
```
Comment (dkayiwa) [inline src/components/bulkConcepts/bulkConceptList.jsx]: Is the deletion of this file required as part of the ticket?

Comment (ozezzy) [reply src/components/bulkConcepts/bulkConceptList.jsx]: Yes, the Text Area  which is the only thing the component renders is now on line 165 of src/components/bulkConcepts/addBulkConcepts.jsx, where the selected search results are displayed.

**Comment by dkayiwa** [comment]
Did you look into the coverage decrease?

**Pull request merged**